internal/cli/url_scheme.go

```diff
@@ -0,0 +1,36 @@
+package cli
+
+import (
+	"fmt"
+	"net/url"
+)
+
+type UrlScheme string
+
+const (
+	httpsScheme UrlScheme = "https"
+	httpScheme  UrlScheme = "http"
+)
+
+func addUrlScheme(urlString string, scheme UrlScheme) (string, error) {
+	parsedUrl, err := url.Parse(urlString)
+	// The url.Parse function may return an error when there's no scheme due to parser ambiguities,
+	// so we need to verify if the error is because the url lacks a scheme
+	if err != nil && parsedUrl.Scheme != "" {
```

Comment:
Mind pulling out the err check and delete the rest?
```suggestion
	if err != nil {
```